docs/examples/lisflood.ipynb

```diff
@@ -5,131 +5,178 @@
    "id": "c02a5bfe",
```

Comment:
<p>Doing a spatial selection might warrant a description in markdown</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eWaterCycle/ewatercycle/pull/175/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
<p>The description is added.</p>